src/main/java/moze_intel/projecte/api/ITileEmc.java

```diff
@@ -2,36 +2,46 @@
 
 /**
  * Interface for TileEntities with an EMC buffer.<br>
- * Note that the EMC should be a double.<br>
  * Changes to the EMC buffer are usually done server-side, so packet handling is required.
  */
-public interface ITileEmc 
+public interface ITileEmc
 {
+
 	/**
 	 * Set the EMC value of the tile entity.<br>
 	 * Gets called only server-side. 
+	 *
+	 * @param value The EMC value
 	 */
-	public void setEmc(double value);
+	public void setEmc(int value);
```

Comment:
Umm what's the reason for this?
EMC is a double internally, and doing this will break binary compatibility (addons compiled earlier expect to see setEmc(double) and then crash when they don't)
I'm sure the mod itself would crash too because it uses this API call quite a bit